code/datums/datacore.dm

```diff
@@ -291,6 +328,46 @@
 		locked += L
 	return
 
+/datum/datacore/proc/generate_security_record(id, mob/living/carbon/human/H)
+	var/datum/data/record/S = new()
+	S.fields["ckey"]		= H.ckey
+	S.fields["id"]			= id
+	S.fields["name"]		= H.real_name
+	S.fields["criminal"]	= "None"
+	S.fields["citation"]	= list()
+	S.fields["crim"]		= list()
```

Comment:
Is this a typo? "Crim"

Reply:
Surprisingly, it is not.

Reply:
Originally, crimes were split into major and minor, and those were called mi_crim and ma_crim. I just left it as crim.